geomesa-blobstore/geomesa-blobstore-handlers/geomesa-blobstore-exif-handler/src/main/scala/org/locationtech/geomesa/blob/handlers/exif/MetadataFileHandler.scala

```diff
@@ -0,0 +1,43 @@
+package org.locationtech.geomesa.blob.handlers.exif
+
+import java.io.File
+import java.lang.Boolean
+import java.util
+import java.util.Date
+
+import com.drew.imaging._
+import com.drew.metadata.exif.GpsDirectory
+import com.vividsolutions.jts.geom.Geometry
+import org.locationtech.geomesa.blob.core.handlers.AbstractFileHandler
+import org.locationtech.geomesa.utils.text.WKTUtils
+
+import scala.collection.JavaConversions._
+
+class MetadataFileHandler extends AbstractFileHandler {
+  override def getGeometryFromFile(file: File): Option[Geometry] = {
+    val gps = ImageMetadataReader.readMetadata(file).getDirectoriesOfType(classOf[GpsDirectory])
+    val gl = gps.head.getGeoLocation
+
+    Option(WKTUtils.read(s"POINT(${gl.getLongitude} ${gl.getLatitude})"))
```

Comment:
@jahhulbert-ccri How do you generate just a Point?  I tend to find it cumbersome, and the WKTUtils were quite handy:)